### PR TITLE
Fixed issue with incorrect dns record set ttl parameter name

### PR DESCRIPTION
**Pull request by MikhailTryakhov**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@MikhailTryakhov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: arm-dns/2016-04-01/swagger/dns.json [hunk 1]
```diff
@@ -937,6 +937,7 @@
         "TTL": {
           "type": "integer",
           "format": "int64",
+	   "x-ms-client-name": "ttl",
```
Comment (kirthik) [inline arm-dns/2016-04-01/swagger/dns.json]: @amarzavery : `x-ms-client-name` is the same as the top level name but lower letters. Based on your talk today afternoon, is this valid? or a no-op?

Comment (amarzavery) [inline arm-dns/2016-04-01/swagger/dns.json]: From the source code [place1](https://github.com/Azure/autorest/blob/master/src/generator/AutoRest.CSharp/CSharpCodeNamer.cs#L167), [place2](https://github.com/Azure/autorest/blob/master/src/core/AutoRest.Core/CodeNamer.cs#L364), it looks that this is a no-op.

Comment (kirthik) [inline arm-dns/2016-04-01/swagger/dns.json]: @MikhailTryakhov : Have you generated the code to test this works? @amarzavery and I looked at the code that would get generated and it looks like in case of `TTL` or `ttl` the code that gets generated is `Ttl`

Comment (MikhailTryakhov) [inline arm-dns/2016-04-01/swagger/dns.json]: Nope, I didn't do it because of test failed, sorry.
Reworked it to eaisier case to make parameter name valid and equals to previous versions

**Comment by kirthik** [comment]
@amarzavery : Node build in travis is failing. Any idea why? It is not obvious from the log. Does not seem related to this PR.

File: arm-dns/2016-04-01/swagger/dns.json [hunk 2]
```diff
@@ -934,7 +934,7 @@
           },
           "description": "Gets or sets the metadata attached to the resource."
         },
-        "TTL": {
+        "ttl": {
```
Comment (kirthik) [inline arm-dns/2016-04-01/swagger/dns.json]: I am a little unclear on what you are trying to achieve. Changing from All Caps to small letters will not change anything during generation. This will still be Pascal cased in C# for instance so `TTL` or `ttl` will be generated as `Ttl`. Could you please take a moment to explain why this change is being made?

Comment (amarzavery) [inline arm-dns/2016-04-01/swagger/dns.json]: Kirthi is correct. The above change will still be a no-op on the generated code.
